Resources/Prototypes/Entities/Mobs/NPCs/pets.yml

```diff
@@ -722,3 +722,26 @@
     attributes:
       proper: true
       gender: male
+
+- type: entity
+  name: Squirt
```

Comment:
squirt 😳😳😳

Reply:
I'm not saying I stole it from Finding Nemo, but I am aware that is a turtles name in Finding Nemo.